internal/ingress/controller/config/config.go

```diff
@@ -77,7 +77,7 @@ const (
 	sslSessionCacheSize = "10m"
 
 	// Default setting for load balancer algorithm
-	defaultLoadBalancerAlgorithm = "least_conn"
+	defaultLoadBalancerAlgorithm = ""
```

Comment:
@diazjf technically it would not matter in our case since we have a guard in `buildLoadBalancingConfig` to return `""` when it is `round_robin`, but I'd like to stick to Nginx configuration. And Nginx does not let you configure it as `round_robin` inside an `upstream` section. You would get:
```
 unknown directive "round_robin"
```
if you do so.